execution_engine/src/runtime/mod.rs

```diff
@@ -2280,7 +2280,13 @@ where
 
         let txn_hash = self.context.get_transaction_hash();
         let from = InitiatorAddr::AccountHash(self.context.get_caller());
-        let fee = Gas::zero(); // TODO
+        let fee = Gas::from(
```

Comment:
this is good but a similar thing needs to be done in the `record_transfer` method of mint_native

Reply:
Done!